### DISCUSSION TITLE
sBTC Design - General Clarity Work

### DISCUSSION THREAD

**Discussion by AshtonStephens** [stacks-archive/sbtc-developer-release#344] [Ideas]
This is a discussion that contains the design document(s) for the overall clarity work to implement for SIP-025.

**Comment by AshtonStephens** [comment]
Currently under investigation by @setzeus.

**Comment by setzeus** [comment]
Draft of design doc here:
https://docs.google.com/document/d/1gf3Ltqge7ChY45kVjf5tNMt8buuoE-WGOj5R_ZIxMi4/edit?usp=sharing

**Comment by AshtonStephens** [comment]
Issue of formalizing document placement can be found here: https://github.com/stacks-network/sbtc/issues/361
